app/javascript/components/app.css

```diff
@@ -1,16 +1,16 @@
 .greeting {
```

Comment:
- [ ] OPTIONAL
Your app does not have the proper git commits. Your Pull Request indicates that you've worked on only two files. Kindly ensure to make the proper commits at appropriate stages of development.